### ISSUE TITLE
Compliance with COPPA

### ISSUE THREAD

**Issue by orionblastar** [orionblastar/K666#38]
In the USA there is a COPPA law that protects children under 13. Scoop did not have this nor does Hacker News and other sites. Each violation is fined $16K USD, not per user but per use. 

It is a simple term of service EULA that the user answers a question that they are 13 or over and we store their response in the database. If under 13, we tell them to get their parents to approve to create their own account to check out our system, and not create an account for the children.

Anyway, we need to think of terms of service TOS and EULA or whatever. I might have written one at some time in the past. We need to be able to follow the law and keep the kiddies safe from the creeper guys or whatever.
[https://www.ftc.gov/enforcement/rules/rulemaking-regulatory-reform-proceedings/childrens-online-privacy-protection-rule](
https://www.ftc.gov/enforcement/rules/rulemaking-regulatory-reform-proceedings/childrens-online-privacy-protection-rule)

I think it is safe to bet that most Kuro5hin/Kr5ddit etc users are 18 or older, and greybeards like Tdillo or Crawford. You never know when some innocent kid stumbles onto one of these troll sites or whatever and sees something they shouldn't, or someone says/writes something to them that they shouldn't.

In other words no kiddie pool, adult swim only.

**Comment by orionblastar** [comment]
Yeah just make a check box that says "I'm over 13" and don't create an account if the box is unchecked and then store it in the user database, so that we know they are legal. If they are 13 and under and checked the box they lied, and their parents can contact us to delete their account or whatever COPPA says.
